src/components/UserActions/UserActivity.ts

```diff
@@ -116,6 +131,19 @@ export class UserActivity extends Component {
         }
     }
 
+    private filterActions(action: UserAction): boolean {
+        const eventValue = action.raw.event_value || '';
+        const eventType = action.raw.event_type || '';
+        if (action.type === UserActionType.Custom) {
+            return (
+                action.type === UserActionType.Custom &&
+                !this.options.customActionsExclude.includes(eventValue) &&
+                !this.options.customActionsExclude.includes(eventType)
+            );
+        }
+        return true;
+    }
```

Comment:
`action.type === UserActionType.Custom` is evaluated twice, the l139 one is redundant and will always return true.
I would split out to keep things tidy IMHO.

```suggestion
    private filterActions(action: UserAction): boolean {
        return action.type !== UserAction.Custom || this.isCustomActionExcluded(action)
    }

    private isCustomActionExcluded(action: UserAction): boolean {
        const eventValue = action.raw.event_value || '';
        const eventType = action.raw.event_type || '';
        return 
            this.options.customActionsExclude.includes(eventValue) &&
            this.options.customActionsExclude.includes(eventType);
    }
```